test/test-execute/exec-personality-ppc64.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test for Personality=ppc64
+
+[Service]
+ExecStart=/bin/sh -c 'echo $(uname -m); exit $(test $(uname -m) = "ppc64" -o $(uname -m) = "ppc64le")'
+Type=oneshot
+Personality=ppc64
```

Comment:
I did that because I wanted to fix the tests on both and the Personality= option didn't seem to have any effect whatsoever.

Also, I would have to add something like this:
```
+#elif defined(__powerpc64__) && defined(__BIG_ENDIAN__)
+        test(m, "exec-personality-ppc64.service", 0, CLD_EXITED);
+#elif ...
```
And I'm not sure if __BIG_ENDIAN__ is supported everywhere. What do you suggest?